src/schema/input.ts

```diff
@@ -34,6 +34,37 @@ export const tmtArtifactsInputSchema = stringToArraySchema.transform(arr => {
 });
 export const tmtContextInputSchema = keyValueArrayToObjectSchema;
 
+// See https://tmt.readthedocs.io/en/stable/spec/plans.html#hardware
+// https://gitlab.com/testing-farm/docs/root/-/merge_requests/120/diffs?view=inline
+export const tmtHardwareSchema = z
```

Comment:
This is definitely not complete, I wonder if we need to validate it here, if so, we should provide rather some npm liberary for validation:

https://gitlab.com/testing-farm/artemis/-/blob/main/server/src/tft/artemis/schema/environment-v0.0.69.yml?ref_type=heads

Could we for now just use an object, and not validate it?

Reply:
I will have a look at it with team. thanks for suggestion. I need to find out solution how to implement it.

Reply:
It should be fixed by https://github.com/sclorg/testing-farm-as-github-action/pull/146/commits/61532115af926ad0d9a5676858afd0c4e7ebd3d1